src/utils/prepare_sorting.tsx

```diff
@@ -0,0 +1,29 @@
+
+
+export const handleSortClick = (column: string , currentSort : string | null , currentOrder : string | null) => {
+  if (currentSort !== column) {
+    return { sort: column};
+  }
+
+  if (currentOrder === '') {
+    return { sort: column, order: 'desc' };
+  }
+
+  return {sort: null , order: null};
+};
+
+export const getSortIcon = (field) => {
+  const sort = field['sort']
+  const order = field['order']
```

Comment:
```suggestion
export const getSortIcon = ({ sort, field ) => {
```
looks like can be shorter.
and function should be typed (arguments and return)